fastlane/Fastfile

```diff
@@ -918,10 +919,14 @@ platform :ios do
 
 end
 
-def push_pods
+def push_revenuecat_pod
   pod_push(path: "RevenueCat.podspec", synchronous: true)
 end
 
+def push_revenuecatui_pod
+  pod_push(path: "RevenueCatUI.podspec", synchronous: true)
```

Comment:
we technically don't need `synchronous` for this one, but it can't hurt to have it